common.css

```diff
@@ -0,0 +1,50 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans&family=Poppins&family=Roboto&family=Roboto+Condensed&display=swap');
+*
+body 
+{
+    font-family: "Roboto", sans-serif;
+    font-size: 1rem;
```

Comment:
By default font-size is 16px. no need to redeclare.

Reply:
Done.